app/controllers/watchers_controller.rb

```diff
@@ -105,20 +105,7 @@ def set_watcher(user, watching)
 
     respond_to do |format|
       format.html { redirect_to :back }
-      format.js do
-        if params[:replace].present?
-          if params[:replace].is_a? Array
-            @replace_selectors = params[:replace]
-          else
-            @replace_selectors = params[:replace].split(',').map(&:strip)
-          end
-        else
-          @replace_selectors = ['#watcher']
-        end
-        @user = user
-
-        render action: 'replace_selectors'
-      end
+      format.js
```

Comment:
the `rjs` temlate is gone, the current implementation uses `watch.js` and `unwatch.js` (as defined by Rails magic) - I am currently looking into merging these two.